sdk/communication/azure-communication-sms/CHANGELOG.md

```diff
@@ -1,17 +1,12 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.4 (2021-09-21)
```

Comment:
Also I do not see any source changes for the two libraries even though the current version was a beta release. Are we sure no sources were changes between the patch releases?

Also, did you use the generate-patch script for these? It is optional to use it but it does verify tag sources etc and I am mainly looking for dogfooding.

Reply:
Since the teams recently took ownership,
I doubt there has been any feature work.

Since the main branch is available for patch release, just used prepare release script in this scenario.
We can use Generate Patch for Communication Identity pacakge PR next.

Reply:
Sounds good.